rocAL/rocAL/source/meta_node_resize.cpp

```diff
@@ -41,32 +41,23 @@ void ResizeMetaNode::update_parameters(MetaDataBatch* input_meta_data)
     _dst_height = _node->get_dst_height();
     for(int i = 0; i < _batch_size; i++)
```

Comment:
The resize node should not do anything since it is a no-op when coordinates are normalized

Reply:
removed the implementation for resize as coordinates are normalized